lang/test/org/partiql/lang/planner/operators/CustomOperatorFactoryTests.kt

```diff
@@ -1,151 +0,0 @@
-package org.partiql.lang.planner.operators
```

Comment:
I am not sure why this got deleted.  Note to self: un-delete this.

Reply:
Are you still going to un-delete this file?

Reply:
oops yes.

Reply:
restored.